src/main/java/com/github/tomakehurst/wiremock/core/WireMockConfiguration.java

```diff
@@ -171,6 +173,13 @@ public WireMockConfiguration timeout(int timeout) {
     return this;
   }
 
+  public WireMockConfiguration serverName(String serverName) {
+
+    this.serverName = serverName;
+    ((Slf4jNotifier) notifier).updateLogger(serverName);
```

Comment:
This unchecked cast impacts extensibility IMO. I suggest doing proper `instanceof` verification

Reply:
done